docs/source/notebooks/SMC-ABC_Lotka-Volterra_example.ipynb

```diff
@@ -80,7 +80,7 @@
    "outputs": [],
```

Comment:
<p>With the goal of gradually standardizing the NBs format, maybe you can use the import statements convention we use on the resources repo? And also use ArviZ style?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>